_health-care/_js/_components/full_name.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import _ from 'lodash';
+
+class FullName extends React.Component {
+  constructor() {
+    super();
+    this.state = {hasError: false};
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentWillMount() {
+    this.id = _.uniqueId();
+  }
+
+  handleChange() {
+    const name = {
+      first: this.refs.first.value,
+      middle: this.refs.middle.value,
+      last: this.refs.last.value,
+      suffix: this.refs.suffix.value
+    }
+
+    const requiredFields = [this.refs.first, this.refs.last];
+
+    for (var i=0; i<requiredFields.length; i++) {
+      let errorDiv = requiredFields[i].parentElement;
+      
+      errorDiv.classList.remove('usa-input-error');
```

Comment:
Maybe move this into the else close below so we don't unnecessarily remove the class?